### PR TITLE
Banner image

### PR DESCRIPTION
**Pull request by sampofilms**
@boveus @CGillen Hey all, I added the banner image to the config. Check configuration.rb line 250 What do you think?

### CODE REVIEW

File: lib/hyrax/configuration.rb [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+    #   @see Hyrax::RightsStatementService for implementation details
+    attr_writer :rights_statement_service_class
+    def rights_statement_service_class
+      @rights_statement_service_class ||= Hyrax::RightsStatementService
+    end
+
+    attr_writer :banner_image
+    def banner_image
+      # This image can be used for free and without attribution. See here for source and license: https://github.com/samvera/hyrax/issues/1551#issuecomment-326624909
+      @banner_image ||= 'https://user-images.githubusercontent.com/20469703/39268224-12874a7e-4884-11e8-9c09-ccef2c0ab28e.png'
```
Comment (sampofilms) [inline lib/hyrax/configuration.rb]: Changed banner_image to png here:

**Comment by boveus** [comment]
@mjgiarlo Is this the recommended way to modify the banner?

**Comment by mjgiarlo** [comment]
@boveus Close, but here's a hint: you should already have a file in your app where you can tweak this value (rather than pasting in `lib/hyrax/configuration.rb`). Ask your friend `grep` to go look for it and tag me when I can take another look. :smiley:

**Comment by mjgiarlo** [comment]
@boveus This looks even better!

**Pull request merged**